src/ansible/roles/facts/tasks/CentOS.yml

```diff
@@ -1,2 +1,19 @@
-- name: 'Facts are the same as in Fedora'
-  include_tasks: 'Fedora.yml'
\ No newline at end of file
+- name: Set distribution specific facts
+  set_fact:
+    systemd:
+      services:
+        kadmin: kadmin.service
+        krb5kdc: krb5kdc.service
+        sshd: sshd.service
+    buildroot: yes
+    debuginfo: yes
+
+- name: Define passkey for CentOS9+
+  set_fact:
+    passkey_support: Yes
+  when: (ansible_distribution_major_version | int >= 9)
+
+- name: Define passkey for CentOS8-
+  set_fact:
+    passkey_support: No
+  when: (ansible_distribution_major_version | int < 9)
```

Comment:
You can keep the Fedora include and then just override what you need (see https://github.com/SSSD/sssd-ci-containers/blob/master/src/ansible/roles/facts/tasks/RedHat.yml). It may also be clearer to separate this to CentOS8.yml and CentOS9.yml.